Cubical/Categories/Profunctor/ComposeFullyFaithful.agda

```diff
@@ -0,0 +1,70 @@
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Profunctor.ComposeFullyFaithful where
```

Comment:
This doesn't have anything specifically to do with Profunctors, should go under Categories.Functor.More probably